spec/hashTable.js

```diff
@@ -0,0 +1,20 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import HashTable from '../src/hashTable'
+
+chai.use(chaiChange)
+
+describe('hashTable', () => {
+  it('exists', () => {
+    expect(HashTable).to.be.a('function')
+  })
+  context('put()', () => {
+    it.only('adds a key-value pair to the hash table.', () => {
+      const ht = new HashTable()
+      ht.put('name', 'sal')
+      console.log('where are you', ht);
+      expect(ht.dataStore.name).to.eql('sal')
```

Comment:
Consider using only the public API of the `HashTable` to test. This is dependent on the underlying representation of data in your `HashTable`, so if the underlying representation changes - even though the implementation still works - all of your tests will fail.